proofs/src/program/manifest.rs

```diff
@@ -82,6 +82,24 @@ pub struct Manifest {
   pub response: ManifestResponse,
```

Comment:
do you think we can move manifest completely out of proofs into a `web-prover-core` as @Autoparallel mentioned, and just keep origo related inputs code in proofs crate?

maybe we can create a origoManifest on top of Manifest separately?

Reply:
I think the separation between Origo and TEE/Proxy manifest makes sense.

Created: https://github.com/pluto/web-prover/issues/490